_docs/how_to_run_acceptance_tests.md

```diff
@@ -0,0 +1,172 @@
+
+_Remember to add your user to the group that allows access to docker/libvirt._
+
+### Bundle/Ruby
+
+Beaker is installed as a gem via bundler, it will run in ruby 2.7 to 3.4 (as of time of writing).
+It should be trivial to install via your package manager or gem.
+
+#### Debian/Ubuntu
+```shell
+# apt install ruby-bundler
+```
+
+#### RedHat (and similar)
+As of el9, dnf replaces yum.  Use the appropriate package manager to get it installed.
+```shell
+# yum install rubygem-bundler
+# dnf install rubygem-bundler
+```
+
+#### ArchLinux
+```shell
+# pacman -Sy extra/ruby-bundler
+```
+
+#### By Gem (Last Option)
+```shell
+# gem install bundler
+```
+
+### Configuring Bundler
+
+Install all needed gems for spec tests into the modules directory by running:
+
+```shell
+bundle config set --local path '.vendor'
+bundle config set --local without 'development release'
+BUNDLE_JOBS="$(nproc)" bundle install
+```
+
+This will install the gems and working space for bundler into a directory called .vendor in the
+module's top level directory.
+If your bundle stops working for some reason, you can run `bundle update && bundle clean` to fix it.
+
+## Setting up to run a test suite
+
+At this point bundler and beaker are installed and ready for use.
+For Vox Pupuli's acceptance testing suite, beaker is managed by a set of environment variables.
+
+
+* `BEAKER_HYPERVISOR` Sets the Hypervisor, `vagrant` or `vagrant_libvirt` for VM based testing
+  It is a good idea to export the `BEAKER_HYPERVISOR` variable in your shell configuration.
+* `BEAKER_DESTROY` Should the test environment be removed at the end of a test
+  * `BEAKER_DESTROY=onpass` Only removes the environment if everything passes, allowing review of the system in the state it was in when the test suite failed.
+  * `BEAKER_DESTROY=no` Always leave an artifact, which likely will break future runs without cleanup.
+  * `BEAKER_DESTROY=yes` _(DEFAULT)_ always clean up.
+* `BEAKER_PROVISION` Should we ensure a clean system is built to run the suite.
+  * If `BEAKER_DESTROY` is set to no, `BEAKER_PROVISION=yes` will fail the run (because of the existing box/container)
+  * If `BEAKER_DESTROY` is set to no, `BEAKER_PROVISION=no` will run the test suite against the system anyways, which may cause environmental issues if the test suite doesn't perfectly put things to a default state.
+
+* `BEAKER_SETFILE` What should we be testing.  Beaker will call beaker-hostgenerator to create a defaultconfiguration based on this, using the known configurations for Vox Pupuli.
+* `BEAKER_PUPPET_COLLECTION` What implementation and version of puppet are we testing against
+
+
+### Listing Platforms
+
+Installed as part of the voxpupuli test suite is `metadata2gha`, which as its name suggests, is a helper to generate a json blob containing the variables for all supported oses.
+
+```shell
+# From the base directory of the module
+$ bundle exec metadata2gha | tail -n 1 | cut -d= -f2- | jq
```

Comment:
I have thought about providing a human readable version on stdout by default and write GitHub output only if that env var is detected.

But in the short term, is better to grep for `puppet_beaker_test_matrix/` instead of relying on `tail`?
```suggestion
$ bundle exec metadata2gha | grep puppet_beaker_test_matrix | cut -d= -f2- | jq
```